README.md

```diff
@@ -239,6 +239,30 @@ The device plugin will initially discover all PCI network resources in the host
 4. "pfNames" - The Physical funtion name
 5. "linkTypes" - The link type of the net device associated with the PCI device.
 
+### Using UDEV Rules for changing NIC's name:
+
+UDEV rules allow the user to change the NIC's name every time the node get rebooted:
+
+1. Add a new rule:
+```
+# vi /etc/udev/rules.d/new.rules
```

Comment:
rules file naming convention is to name the file starting with a priority e.g : 90-netnames.rules